nipype/pipeline/plugins/tests/test_sgelike.py

```diff
@@ -0,0 +1,45 @@
+from nipype.pipeline.plugins.base import SGELikeBatchManagerBase
+from nipype.interfaces.utility import Function
+import nipype.pipeline.engine as pe
+from os.path import join
+import os
+from glob import glob
+import pytest
+from mock import patch
```

Comment:
We should be able to use the builtin `unittest.mock`.

```suggestion
from unittest.mock import patch
```